bbn-1/finality-providers/README.md

```diff
@@ -0,0 +1,377 @@
+New key for the BTC chain is created (mnemonic should be kept in a safe place for recovery):
+{
+  "name": "my-key-name",
+  "pub_key_hex": "50b106208c921b5e8a1c45494306fe1fc2cf68f33b8996420867dc7667fde383",
+  "mnemonic": "bad mnemonic private tilt wish bulb miss plate achieve manage feel word safe dash vanish little miss hockey connect tail certain spread urban series"
+}
+```
+
+> Reminder: if the `--keyring-backend` flag was used to create the key, the
+same flag should be used later for accessing this key.
+
+At the end of these steps, the EOTS BTC key pair will be generated. The key
+pair or the mnemonic generated must be stored in a safe place, as it is
+expected to be needed for the finality provider's participation in PoS security
+in the future stages of the Babylon mainnet. Finality providers that don't have
+access to their keys, will not be able to transition to later stages.
+
+**⚠ Warning!**
+Store the **mnemonic** in a safe place. The mnemonic is the only way you can
+recover your keys in the case of loss or file system corruption.
+
+### 3. Create your Finality Provider Information object
+
+After [forking](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
+the current repository, navigate to the `bbn-1/finality-providers` directory
+and create a file under the `finality-providers/registry/${nickname}.json`
+path. `${nickname}`, corresponds to a unique human-readable nickname your
+finality provider can be identified with (e.g. your moniker). It should not
+contain white spaces or unrecognizable characters.
+
+Inside this file, store the following JSON information corresponding to the
+finality provider.
+
+```json
+{
+  "description": {
+    "moniker": "<moniker>",
+    "identity": "<identity>",
+    "website": "<website>",
+    "security_contact": "<security_contact>",
+    "details": "<details>"
+  },
+  "eots_pk": "<finality_provider_eots_pk>",
+  "commission": "<commission_decimal>"
+}
+```
+
+Properties descriptions:
+
+- `moniker`: nickname of the finality provider.
+- `identity`: optional Keybase.io identity signature. It is used to retrieve
+the finality provider icon, the same way Cosmos-SDK uses for validators.
+- `website`: optional website link URL.
+- `security_contact`: required email for security contact.
+- `details`: any other optional detail information.
+- `eots_pk`: the btc pub key in hex format (The `pub_key_hex` property of the
+`eotsd keys add` command output).
+- `commission`: the commission charged for BTC staking rewards.
+The commission will be parsed as a decimal:
+  - `"1.00"` represents 100% commission.
+  - `"0.10"` represents  10% commission.
+  - `"0.01"` represents  01% commission.
+
+**⚠ Warning!**
+The minimum commission value accepted is 5%. It will remain immutable and can't
```

Comment:
Let's wait until the decision for this is finalized.

Reply:
3%, I will update

Reply:
https://github.com/babylonlabs-io/networks/pull/1/commits/502fc3f824d3a163448dcdcea3fe89e29989b144